build.gradle

```diff
@@ -107,3 +107,21 @@ sonarqube {
         }
     }
 }
+
+/**
+ * Task that will call the checkStyle task fpr all subproject and included build.
```

Comment:
nit: fpr -> for?

What do you mean the checkstyle will be called for the included build? can you also explain why this is needed?

Reply:
Thanks for spotting it, I also spotted it later, and fixed it with this [commit](https://github.com/bitrise-io/trace-android-sdk/pull/204/commits/735fccfdd53c4cd30afc0d2e4206327c509af34a) if it is ok.

Currently trace-android-sdk have 2 subprojects, trace-sdk and trace-test-applicaiton, and 3 included builds, buildSrc, pencensus and trace-gradle-plugin. 
The purpose of this is to be an umbrella task, that will call checkStyle tasks in all subprojects and included builds.